pytorch_lightning/loops/optimization/manual_loop.py

```diff
@@ -149,3 +145,17 @@ def _on_before_step(self) -> None:
     def _on_after_step(self) -> None:
         self.trainer.profiler.stop("optimizer_step")
         self.optim_step_progress.increment_completed()
+
+    def _build_kwargs(self, kwargs: OrderedDict, hiddens: Optional[Any]) -> OrderedDict:
```

Comment:
why the indirection of this method, if we could just directly call the function?
is this meant to be overridden for customization?

Reply:
Yes, and for consistency. Although I realize that it's prefixed by `_`

I'll resolve this comment but tell me if you prefer to remove it. I don't mind.